src/constants/postFom.ts

```diff
@@ -0,0 +1,169 @@
+    { title: '6', value: '6' },
+    { title: '7', value: '7' },
+    { title: '8', value: '8' },
+    { title: '9', value: '9' },
+    { title: '10', value: '10' },
+  ],
+};
+
+export const SCRIPT_URL = '//t1.daumcdn.net/mapjsapi/bundle/postcode/prod/postcode.v2.js';
+
+export const addressCustomTheme = {
+  bgColor: '#0D0D0D',
+  searchBgColor: '#0D0D0D',
+  contentBgColor: '#141414',
+  pageBgColor: '#141414',
+  textColor: '#ACACAC',
+  queryTextColor: '#ACACAC',
+  postcodeTextColor: '#852DFF',
+  emphTextColor: '#852DFF',
+  outlineColor: '#ACACAC',
+};
+
+export const schedulesTime = {
+  start: [
+    { title: '00:00', value: '00:00' },
+    { title: '01:00', value: '01:00' },
+    { title: '02:00', value: '02:00' },
+    { title: '03:00', value: '03:00' },
+    { title: '04:00', value: '04:00' },
+    { title: '05:00', value: '05:00' },
+    { title: '06:00', value: '06:00' },
+    { title: '07:00', value: '07:00' },
+    { title: '08:00', value: '08:00' },
+    { title: '09:00', value: '09:00' },
+    { title: '10:00', value: '10:00' },
+    { title: '11:00', value: '11:00' },
+    { title: '12:00', value: '12:00' },
+    { title: '13:00', value: '13:00' },
+    { title: '14:00', value: '14:00' },
+    { title: '15:00', value: '15:00' },
+    { title: '16:00', value: '16:00' },
+    { title: '17:00', value: '17:00' },
+    { title: '18:00', value: '18:00' },
+    { title: '19:00', value: '19:00' },
+    { title: '20:00', value: '20:00' },
+    { title: '21:00', value: '21:00' },
+    { title: '22:00', value: '22:00' },
+    { title: '23:00', value: '23:00' },
+  ],
+
+  end: [
+    { title: '01:00', value: '01:00' },
+    { title: '02:00', value: '02:00' },
+    { title: '03:00', value: '03:00' },
+    { title: '04:00', value: '04:00' },
+    { title: '05:00', value: '05:00' },
+    { title: '06:00', value: '06:00' },
+    { title: '07:00', value: '07:00' },
+    { title: '08:00', value: '08:00' },
+    { title: '09:00', value: '09:00' },
+    { title: '10:00', value: '10:00' },
+    { title: '11:00', value: '11:00' },
+    { title: '12:00', value: '12:00' },
+    { title: '13:00', value: '13:00' },
+    { title: '14:00', value: '14:00' },
+    { title: '15:00', value: '15:00' },
+    { title: '16:00', value: '16:00' },
+    { title: '17:00', value: '17:00' },
+    { title: '18:00', value: '18:00' },
+    { title: '19:00', value: '19:00' },
+    { title: '20:00', value: '20:00' },
+    { title: '21:00', value: '21:00' },
+    { title: '22:00', value: '22:00' },
+    { title: '23:00', value: '23:00' },
+    { title: '24:00', value: '24:00' },
+  ],
+};
+
+export const VALID_IMAGE_URL = {
+  usual: 'https://sprint-fe-project.s3.ap-northeast-2.amazonaws.com/globalnomad/activity_registration_image/',
+  unusual: 'https://sprint-fe-project.s3.ap-northeast-2.amazonaws.com/globalnomad/activity_registration_image',
+};
+
+export const DEFAULT_API_DATA = { address: '서울 중구 청계천로 100', bannerImageUrl: PNGS.banner.BATTLEGROUNDS.url };
```

Comment:
오프라인 주소를 제외하고 아예 address 안에 '온라인'을 default로 넣어주면 어떨까요?
```suggestion
export const DEFAULT_API_DATA = { address: '온라인', bannerImageUrl: PNGS.banner.BATTLEGROUNDS.url };
```